test/src/test/java/org/corfudb/integration/cluster/Harness/Harness.java

```diff
@@ -0,0 +1,103 @@
+package org.corfudb.integration.cluster.Harness;
+
+import org.corfudb.integration.AbstractIT;
+import org.corfudb.runtime.BootstrapUtil;
+import org.corfudb.runtime.view.Layout;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+import static org.corfudb.AbstractCorfuTest.PARAMETERS;
+import static org.corfudb.integration.AbstractIT.getCorfuServerLogPath;
+
+/**
+ * Utilities to form and manipulate a corfu cluster.
+ * <p>
+ * <p>Created by maithem on 7/18/18.
+ */
+
+public class Harness {
+
+    String localAddress = "localhost";
+
+    final int basePort = 9000;
+
+    private String getAddressForNode(int port) {
+        return "localhost" + ":" + port;
+    }
+
+    /**
+     * Creates an n-node cluster layout
+     * @param n number of nodes in the cluster
+     * @return a layout for the cluster
+     */
+    private Layout getLayoutForNodes(int n) {
+        List<String> layoutServers = new ArrayList<>();
+        List<String> sequencer = new ArrayList<>();
+        List<Layout.LayoutSegment> segments = new ArrayList<>();
+        UUID clusterId = UUID.randomUUID();
+        List<String> stripServers = new ArrayList<>();
+
+        long epoch = 0;
+
+        for (int x = 0; x < n; x++) {
+            layoutServers.add(getAddressForNode(x));
```

Comment:
is the port intended to be derived from basePort? getAddressForNode(basePort + x)

Reply:
Yes, good catch. Addressed in #1385